FreeRTOS-Plus/Source/FreeRTOS-Plus-TCP/FreeRTOS_DNS.c

```diff
@@ -1470,14 +1548,18 @@ BaseType_t xReturn;
 	{
 	BaseType_t x;
 	BaseType_t xFound = pdFALSE;
-	uint32_t ulCurrentTimeSeconds = ( xTaskGetTickCount() / portTICK_PERIOD_MS ) / 1000;
+	uint32_t ulCurrentTimeSeconds = ( xTaskGetTickCount() / portTICK_PERIOD_MS ) / 1000U;
 	static BaseType_t xFreeEntry = 0;
-		configASSERT(pcName);
+
+		/* MISRA advisory rule 1.2 Relaxed in case of 
+		 * configASSERT as using __FUNCTION__ makes 
```

Comment:
What does this mean?

Reply:
configASSERT uses __FILE__, __LINE__ and __FUNCTION__ in logging to show where exactly the error is. MISRA advises against that.
Since in the later files I stopped commenting for configASSERT, I removed this comment as well.